app/models/observation.rb

```diff
@@ -575,7 +577,7 @@ def self.export_formatted(notes, markup = nil)
     return "" if notes.blank?
     return notes[other_notes_key] if notes.keys == [other_notes_key]
 
-    result = notes.each_with_object("") do |(key, value), str|
+    result = notes.each_with_object(+ "") do |(key, value), str|
```

Comment:
Uses a mutable string. I don't see an easy way to use a frozen string, and that also doesn't make sense to me.

Reply:
This seems reasonable to me.  Does rubocop have an opinion between having a space between `+` and the first quote or not?  Personally I lean towards no space.  I think of it kind of like `-5`.  I assume it has very high precedence since unary operators usually do, so I like to keep them closer.